.github/workflows/lint-test-chart.yaml

```diff
@@ -9,33 +9,36 @@ jobs:
   lint-test:
     if: github.repository == 'kubernetes-sigs/external-dns'
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
```

Comment:
Should this be @v3 as it is in other yaml in this directory?

Reply:
Upstream dependabot migrated it to `v3` [first](https://github.com/kubernetes-sigs/external-dns/commit/edf144ef3efc9259b622ba5b3d46d1dd27a47aee) and then [this PR](https://github.com/kubernetes-sigs/external-dns/pull/2940) came in and decided to pin to a not floating tag but only for the chart related github actions. I think we can keep it as is because 1) it matches the upstream, 2) we don't use the github actions downstream.

Reply:
Is it a problem that the Helm chart (v0.12.2) is not using the same external-dns version that you are rebasing to (v0.13.1)?

Reply:
No, the helm chart is not used by the ExternalDNS Operator in any way.